topo/counting/faucet.yaml

```diff
@@ -0,0 +1,181 @@
+dps:
+  nz-kiwi-t1sw1:
+    dp_id: 177
+    faucet_dp_mac: 0e:00:00:00:01:01
+    hardware: Generic
+    lacp_timeout: 5
+    stack:
+      priority: 1
+    interfaces:
+      6:
+        description: "to t1sw2 port 6"
+        stack: {dp: nz-kiwi-t1sw2, port: 6}
+      9:
+        description: "to t2sw1 port 50"
+        stack: {dp: nz-kiwi-t2sw1, port: 50}
+      10:
+        description: "to t2sw2 port 50"
+        stack: {dp: nz-kiwi-t2sw2, port: 50}
+      28:
+        description: egress
+        lacp: 3
+        tagged_vlans: [100, 200, 300, 400]
+    lldp_beacon: {max_per_interval: 5, send_interval: 5}
+  nz-kiwi-t1sw2:
+    dp_id: 178
+    faucet_dp_mac: 0e:00:00:00:01:02
+    hardware: Generic
+    lacp_timeout: 5
+    stack:
+      priority: 1
+    interfaces:
+      6:
+        description: "to t1sw1 port 6"
+        stack: {dp: nz-kiwi-t1sw1, port: 6}
+      9:
+        description: "to t2sw1 port 52"
+        stack: {dp: nz-kiwi-t2sw1, port: 52}
+      10:
+        description: "to t2sw2 port 52"
+        stack: {dp: nz-kiwi-t2sw2, port: 52}
+      28:
+        description: egress
+        lacp: 3
+        tagged_vlans: [100]
+    lldp_beacon: {max_per_interval: 5, send_interval: 5}
+  nz-kiwi-t2sw1:
+    dp_id: 147058200627
+    faucet_dp_mac: 0e:00:00:00:02:01
+    hardware: Generic
+    lacp_timeout: 5
+    interfaces:
+      1: {description: IoT Device, native_vlan: 100, max_hosts: 1}
+      2: {description: IoT Device, native_vlan: 100, max_hosts: 1}
+      3: {description: IoT Device, native_vlan: 100, max_hosts: 1}
+      4: {description: IoT Device, native_vlan: 100, max_hosts: 1}
+      5: {description: IoT Device, native_vlan: 100, max_hosts: 1}
+      6: {description: IoT Device, native_vlan: 100, max_hosts: 1}
```

Comment:
are all of these necessary for your test?  If not I'd trim them down because all they're doing then is making life more complicated for us!

Reply:
Removed all the unnecessary host ports.